src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -244,4 +243,10 @@ fi
 			fi
 		fi
 	{{ end }}
-{{ end }}
\ No newline at end of file
+{{ end }}
+
+if [ ! -z "$RUN_BUILD_COMMAND" ]
+then
+	echo 'Running $RUN_BUILD_COMMAND'
```

Comment:
nit: just to make the log a little more clear, I would suggest changing the wording here to:

`echo 'Running the following provided RUN_BUILD_COMMAND: $RUN_BUILD_COMMAND`